apiserver/facades/client/subnets/subnets_test.go

```diff
@@ -121,6 +121,17 @@ func (s *SubnetsSuite) setUpMocks(c *gc.C) *gomock.Controller {
 
 	s.mockNetworkService = subnets.NewMockNetworkService(ctrl)
 
+	var err error
+	s.facade, err = subnets.NewAPIWithBacking(
+		&stubBacking{apiservertesting.BackingInstance},
+		apiservertesting.NoopModelCredentialInvalidatorGetter,
+		s.resources,
+		s.authorizer,
+		loggertesting.WrapCheckLog(c),
+		s.mockNetworkService,
+	)
+	c.Assert(err, jc.ErrorIsNil)
+	c.Assert(s.facade, gc.NotNil)
```

Comment:
This isn't really a 'mock' being set up, so this is a little misleading, but it's not enough of a problem for me to really care

Reply:
Yea, I've had this problem in a few places before. In the past I've added an extra `setUpFacade` method, but then I saw this pattern elsewhere and realised its not really worth polluting the first two lines of every test with two separate methods